CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required (VERSION 2.8.8)
+cmake_minimum_required (VERSION 3.0)
```

Comment:
Debian Wheezy uses [cmake 2.8.9-1](https://packages.debian.org/wheezy/cmake).

Seems testmod_date even failed before this PR on wheezy. Otherwise Elektra compiles and all test cases pass, so we should consider to do the version pump after the release. (Then we can also remove the wheezy build agents)

The warnings you get are likely only because of problems within Qt-CMake---nothing we should worry about. We do not use `COMPILE_DEFINITIONS_*`. Workaround: set the policy within `qt-gui`.